src/assets/Main.js

```diff
@@ -65,3 +65,8 @@ function gerenciador() {
     });
 }
 gerenciador();
+
```

Comment:
every single file is lowcase and you choosed to keep those 2 file CamelCase!

change to lowcase as well!